docs/user-guide/installation/f24+.rst

```diff
@@ -0,0 +1,282 @@
+      $ sudo systemctl start pulp_celerybeat
+
+   Lastly, exactly one ``pulp_resource_manager`` process must be running in the installation. This
+   process acts as a task router, deciding which worker should perform certain types of tasks.
+   Apologies for the repetitive message, but it is important that this process only be enabled on
+   one host. Edit ``/etc/default/pulp_resource_manager`` to your liking. Then::
+
+      $ sudo systemctl enable pulp_resource_manager
+      $ sudo systemctl start pulp_resource_manager
+
+   .. warning::
+
+      ``pulp_resource_manager`` must be singleton, so be sure that you
+      only enable this on one host if you are deploying Pulp in a clustered environment.
+
+
+Admin Client
+------------
+
+The Pulp Admin Client is used for administrative commands on the Pulp server,
+such as the manipulation of repositories and content. The Pulp Admin Client can
+be run on any machine that can access the Pulp server's REST API, including the
+server itself. It is not a requirement that the admin client be run on a machine
+that is configured as a Pulp consumer.
+
+Pulp admin commands are accessed through the ``pulp-admin`` script.
+
+
+#. Install the Pulp admin client extentions for the plugin types you wish to use. They depend on
+   pulp-admin itself so you will get that along with them. The following example installs all the
+   currently available admin extensions, feel free to season to taste:
+
+   ::
+
+      $ sudo dnf install pulp-docker-admin-extensions pulp-puppet-admin-extensions pulp-rpm-admin-extensions pulp-ostree-admin-extensions pulp-python-admin-extensions
+
+#. Update the admin client configuration to point to the Pulp server. Keep in mind
+   that because of SSL verification this should be the fully qualified name of the server,
+   even if it is the same machine (localhost will not work with the default apache
+   generated SSL certificate). Regardless, the "host" setting below must match the
+   "CN" value of the server's HTTP SSL certificate.
+   This change is made globally to the ``/etc/pulp/admin/admin.conf`` file, or
+   for one user in ``~/.pulp/admin.conf``:
+
+   ::
+
+      [server]
+      host = localhost.localdomain
+
+
+Consumer Client and Agent
+-------------------------
+
+The Pulp Consumer Client is present on all systems that wish to act as a consumer
+of a Pulp server. The Pulp Consumer Client provides the means for a system to
+register and configure itself with a Pulp server. Additionally, the Pulp Consumer
+Client runs an agent that will receive messages and commands from the Pulp server.
+
+Pulp consumer commands are accessed through the ``pulp-consumer`` script. This
+script must be run as root to permit access to add references to the Pulp server's
+repositories.
+
+#. Install the Gofer bindings for the message broker you are using. For qpidd::
+
+      $ sudo dnf install python-gofer-qpid
+
+   For RabbitMQ::
+
+      $ sudo dnf install python-gofer-amqp
```

Comment:
Yeah that was one of my main goals with this. There are those who want Pulp to support Redis as well…